### ISSUE TITLE
Keyboard focus improvements/follow up.

### ISSUE THREAD

**Issue by yzen** [mozilla-mobile/mozilla-vpn-client#226]
- Focus outline for Log Out should be red (similar to delete device button)
- Focus styling for Device List s sometimes missing:
  - Go to My Devices
  - Make sure you have an item that has a delete button
  - only tab (forward) from back button to list to delete as long as needed until you get to back button again
  - tab (forward) to the list, notice that the list is not highlighted focused, but it actually is..
